<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.200" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2025-04-23T15:21:16" tests="1" file="cypress/e2e/features/assignment_viewing.feature" time="0.198" failures="1">
    <testcase name="An uncaught error was detected outside of a test" time="0.000" classname="An uncaught error was detected outside of a test">
      <failure message="The following error originated from your test code, not from Cypress.

  &gt; Expected to find a global registry (this usually means you are trying to define steps or hooks in support/e2e.js, which is not supported) (this might be a bug, please report at https://github.com/badeball/cypress-cucumber-preprocessor)

When Cypress detects uncaught errors originating from your test code it will automatically fail the current test.

Cypress could not associate this error to any specific test.

We dynamically generated a new test to display this failure." type="Error"><![CDATA[Error: The following error originated from your test code, not from Cypress.

  > Expected to find a global registry (this usually means you are trying to define steps or hooks in support/e2e.js, which is not supported) (this might be a bug, please report at https://github.com/badeball/cypress-cucumber-preprocessor)

When Cypress detects uncaught errors originating from your test code it will automatically fail the current test.

Cypress could not associate this error to any specific test.

We dynamically generated a new test to display this failure.
    at createError (http://localhost:8080/__cypress/tests?p=cypress/support/e2e.js:536:16)
    at fail (http://localhost:8080/__cypress/tests?p=cypress/support/e2e.js:540:15)
    at assert (http://localhost:8080/__cypress/tests?p=cypress/support/e2e.js:547:9)
    at assertAndReturn (http://localhost:8080/__cypress/tests?p=cypress/support/e2e.js:551:9)
    at getRegistry (http://localhost:8080/__cypress/tests?p=cypress/support/e2e.js:9571:49)
    at defineBefore (http://localhost:8080/__cypress/tests?p=cypress/support/e2e.js:10613:38)
    at eval (http://localhost:8080/__cypress/tests?p=cypress/support/e2e.js:11265:51)
    at eval (http://localhost:8080/__cypress/tests?p=cypress/support/e2e.js:11278:3)
    at eval (<anonymous>)]]></failure>
    </testcase>
  </testsuite>
</testsuites>